anchor/database/src/tests/metadata_tests.rs

```diff
@@ -31,8 +31,8 @@ mod tests {
         let metadata = queries::get_metadata(&conn).expect("Failed to get metadata");
 
         assert_eq!(
-            metadata.schema_version, 0,
-            "Initial schema version should be 0"
+            metadata.schema_version, 1,
+            "Initial schema version should be 1"
```

Comment:
Do we need to change it? Apart from this, I think everything else is backwards compatible?

Reply:
But I guess it's not a big deal to require the DB to be deleted for now